docker/Dockerfile

```diff
@@ -5,4 +5,10 @@ COPY docker/entry.sh /usr/bin/entry.sh
 
 ENTRYPOINT ["/usr/bin/entry.sh"]
 
+# Healthcheck is done by checking if the directory is modified within the
+# half of the period given for the update period.
+HEALTHCHECK --interval=10s --timeout=5s \
+    CMD test $(stat -c %Y /usr/share/GeoIP) -gt $(($(date +%s) - $GEOIPUPDATE_FREQUENCY * 60 * 60 - 20 )) \
```

Comment:
This seems reasonable. I wonder if the we should subtract more than 20 seconds though. My concern is that it that there could be false alerts if `geoipupdate` is slow to start or something.

Also, I assume this only works if `LockFile` is not set to something other than the database directory (as otherwise, we wouldn't modify the directory if there are no updates). Is that correct? If so, that should be mentioned in the documentation.